sdk/monitor/Azure.Monitor.Query/src/LogsQueryClient.cs

```diff
@@ -81,7 +81,7 @@ protected LogsQueryClient()
         /// <summary>
         /// Executes the logs query.
         /// </summary>
-        /// <param name="workspace">The workspace to include in the query.</param>
+        /// <param name="workspace">The workspace id to include in the query (<c>xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx</c>).</param>
```

Comment:
should we name this `workspaceId` ?

Reply:
Will this always be a GUID?

Reply:
The service defines the parameter as a string so no guarantees although I doubt it would change.

Reply:
It's system defined though, correct? The only reason I was asking is that we have examples of services that take a any string value, as long as that string is a valid GUID 😄 . In that case we just typed it as GUID to avoid pain.

Reply:
Good point. I'll chat with the team about it.